### ISSUE TITLE
Unable to read language.json file

### ISSUE THREAD

**Issue by lixh00** [golang-module/carbon#139]
Hello,

I encountered an issue with the following code:
```go
lange := carbon.NewLanguage()
lange.SetLocale("zh-CN")
car := carbon.SetLanguage(lange)
if car.Error != nil {
	log.Printf("Error Info %v\n", car.Error)
	return
}

offDutyTime := car.Now().StartOfDay().AddHours(18)
now := car.Now()
diff:= now.DiffInString(offDutyTime)
log.Printf("Diff: %v", diff)
```

golang version: **such as 1.16**
`1.18.1`

carbon version: **such as 1.5.0**
`v2.1.8`

time zone: **such as Japan**
`China`

I expected to get:

```
`Diff: 9小时`
```

But I actually get:

```
invalid locale file "lang\\zh-CN.json", please make sure the json file exists and is valid
```

Pic:
![image](https://user-images.githubusercontent.com/72033265/173470961-8d5176c0-f573-4eb3-a6f5-c90116b58e13.png)
![image](https://user-images.githubusercontent.com/72033265/173470893-49885874-3b14-4cad-b1c8-f97768cee305.png)

Thanks!

**Comment by gouguoyin** [comment]
Sorry, I haven't tested it on the window platform, I will try to find a window platform device to test.

**Comment by chanyipiaomiao** [comment]
I had the same problem， windows10, go.1.18.3, carbon: v2.1.8
`func TestWeek(t *testing.T) {
	lang := carbon.NewLanguage()
	lang.SetLocale("zh-CN")
	c := carbon.SetLanguage(lang)
	if c.Error != nil {
		// 错误处理
		t.Error(c.Error)
	}
	fmt.Println(c.ToWeekString())
}
`
=== RUN   TestWeek
    a_test.go:15: invalid locale file "lang\\zh-CN.json", please make sure the json file exists and is valid

--- FAIL: TestWeek (0.00s)

FAIL

**Comment by gouguoyin** [comment]
It has been fixed in v1.7.1 or v2.2.1
